catalog/transformer/catalog-transformer-csv/src/test/java/ddf/catalog/transformer/csv/CsvQueryResponseTransformerTest.java

```diff
@@ -86,7 +86,7 @@ public void testCsvQueryResponseTransformer() throws CatalogTransformerException
     String[] hiddenFieldsArray = {"attribute3", "attribute5"};
     argumentsMap.put("hiddenFields", buildSet(hiddenFieldsArray));
 
-    String[] columnOrderArray = {"attribute4", "attribute2"};
+    String[] columnOrderArray = {"attribute4", "attribute2", "attribute1"};
```

Comment:
❓Does this test actually verify the new behavior? Since these are the only three attributes that could be exported after removing the object/binary attributes and hidden fields,  wouldn't leaving this as `{"attribute4", "attribute2"}` and changing `expectedHeaders` to `{"attribute4", "column2"}` verify that only the visible attributes are exported?

Reply:
That's true. I've updated the logic b00d6851090174430950fb6deb4a4e39d9d4b5fd